src/Init/Data/Int/DivMod/Lemmas.lean

```diff
@@ -2894,6 +2894,73 @@ theorem bmod_eq_self_of_le_mul_two {x : Int} {y : Nat} (hle : -y ≤ x * 2) (hlt
     x.bmod y = x :=
   bmod_eq_of_le_mul_two hle hlt
 
+/- ### ediv -/
+
+theorem ediv_lt_self_of_pos_of_one_ne_one {x y : Int} (hx : 0 < x) (hy : y ≠ 1) :
```

Comment:
```suggestion
theorem ediv_lt_self_of_pos_of_ne_one {x y : Int} (hx : 0 < x) (hy : y ≠ 1) :
```

I think it should be `of_ne_one` since we are saying that `(hy : y ≠ 1)`, not `1 ≠ 1`.

Reply:
thanks for the catch!! it is indeed, fixed now.